EECS3311-PD1/src/main/java/Analyzer/AnalysisResult.java

```diff
@@ -0,0 +1,90 @@
+package Analyzer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AnalysisResult {
+    private Map<String, Double> firstData;
+
+    private Map<String, Double> secondData;
+
+    private Map<String, Double> thirdData;
```

Comment:
Not a very scalable solution. :)
What if we want to add an analysis that uses 4 types of data? And a month later one that uses 5?
And then one that uses 10?